tests/fixtures.ts

```diff
@@ -51,7 +51,7 @@ export class CodeEditors {
     replacePreviousContent = true,
   }: {nth?: number; text: string; replacePreviousContent?: boolean}) {
     await writeInMonaco({
-      locator: this.codeEditors.nth(nth),
+      locator: this.codeEditors.nth(nth + 1),
```

Comment:
This shouldn't have the plus 1 right?

Reply:
![image](https://github.com/user-attachments/assets/ef431ebc-5b95-475f-9d70-4a8a597b392c)
github foldind is misleading

Reply:
makes sense, sorry